src/js/components/Button/doc.js

```diff
@@ -274,6 +274,10 @@ export const themeDoc = {
     description: `The color of the label for default buttons.`,
     type: 'string | { dark: string, light: string }',
   },
+  'button.default.font.weight': {
+    description: `The weight of the text label for default buttons.`,
+    type: 'string',
```

Comment:
I think this should accept a number as well, can you verify?

Reply:
yes I will verify I just added a number in a string but I will do a number

Reply:
you are right I just checked with 
```
      font: { weight: 800 }
```